addons/mail/static/src/components/message/message.js

```diff
@@ -104,6 +104,9 @@ class Message extends Component {
     _constructor() {}
 
     mounted() {
+        [...this._contentRef.el.querySelectorAll(':scope .o_Message_readMore')].map(
+            el => el.remove()
+        );
```

Comment:
Could be a for-of look instead of `.map()`. Fix looks appropriate, read more/less is legacy code that mutates the DOM, so hard to do much better without re-thinking how read more/less work (waste of time right now).